apis/Google.Cloud.Spanner.Data/run-emulator-tests.sh

```diff
@@ -1,67 +0,0 @@
-#!/bin/bash
```

Comment:
Is it useful to keep this (possibly slimmed down a bit, now that we create the instance ourselves) for the sake of running locally?